### ISSUE TITLE
Build with Font Awesome Pro failed: aliases.xml not found

### ISSUE THREAD

**Issue by AndrasJanurik** [FlowingCode/FontAwesomeIronIconset#83]
Hi, I followed your guide to integrate with Font Awesome Pro but the build failed.
I am using Font Awesome Pro 5.15.4, and I copied the sprites folder according to the guide.
Can you please help?

This is the build error:
[INFO] --- exec:3.1.0:java (generate) @ iron-icons-codegen ---
Sprites directory is D:\Prg\work\EclipseJavaProjects\Others\FontAwesomeProIronIconset\src\main\sprites
Output sources directory is D:\Prg\work\EclipseJavaProjects\Others\FontAwesomeProIronIconset\src\main\generated
Output resources directory is D:\Prg\work\EclipseJavaProjects\Others\FontAwesomeProIronIconset\src\main\resources\META-INF\resources\frontend\font-awesome-iron-iconset
Generate Regular icons (far)
Error at char 9 in expression in xsl:variable/@select on line 75 column 67
  FODC0002  I/O error reported by XML parser processing
  file:///D:/Prg/work/EclipseJavaProjects/Others/FontAwesomeProIronIconset/src/main/sprites/aliases.xml. Caused by java.io.FileNotFoundException: D:\Prg\work\EclipseJavaProjects\Others\FontAwesomeProIronIconset\src\main\sprites\aliases.xml (The system cannot find the file specified)
  In template rule with match="/svg:svg" on line 31 of
     invoked by built-in template rule (text-only)

**Comment by javier-godoy** [comment]
Hello,

In order to generate Pro icons with aliases, you'll need to add the `icons.json` files in the sprites directory (src/main/sprites).
Aliases provides backward compatibility in case you upgrade the version of FontAwesome, see #50 for details.

The `icons.json` file for the free version is available at https://github.com/FortAwesome/Font-Awesome/tree/6.x/metadata
(I think there should be an equivalent for FontAwesome Pro, but I cannot check because I don't have access to FontAwesome Pro sources; if you can confirm, I'll update the build instructions). 

There was also a bug that assumed that `icons.json` was always available. The fix has been merged to `master` minutes ago.

**Comment by AndrasJanurik** [comment]
Hi,

Thanks for your answer.
I wasn't planning on using aliases, I am not even sure what they are or I need them at all. I was just following the build guide and that was the error message I got.

Using your latest code the build progressed further, generated a very short aliases.xml file, and finally failed at compiling the demo files.

I found the icons.json file in my pro package and copied it to the sprites folder. It did not make any difference, the generated aliases.xml was the same and the build failed.

The errors in the demo files were for specific missing icons. I fixed it and then the build succeeded.

Then I updated my project to use the new font awesome pro dependecy. Application compile and build is fine but now I get runtime errors.

I attach some files for you:
- the generated aliases.xml
- the errors related to the demo files
- my pro package file list for you reference
- the runtime error I am getting now

Thanks for the help.
[aliases_xml.txt](https://github.com/FlowingCode/FontAwesomeIronIconset/files/11381228/aliases_xml.txt)
[demo_errors.txt](https://github.com/FlowingCode/FontAwesomeIronIconset/files/11381229/demo_errors.txt)
[pro_package_file_list.txt](https://github.com/FlowingCode/FontAwesomeIronIconset/files/11381230/pro_package_file_list.txt)
[runtime_error.txt](https://github.com/FlowingCode/FontAwesomeIronIconset/files/11381232/runtime_error.txt)
